src/LoginPage.tsx

```diff
@@ -1,11 +1,56 @@
 import * as React from 'react';
+import 'semantic-ui-css/semantic.min.css';
```

Comment:
I don't think you have to include this file here, if it is included in index.tsx.

Reply:
I leave it at a comment, in case of. I'll delete the line later.